packages/blade/src/components/Typography/Text/__tests__/Text.ssr.test.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * @jest-environment node
+ */
+import React from 'react';
+import { Text } from '../';
+import renderWithSSR from '~src/_helpers/testing/renderWithSSR.web';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+describe('<Text />', () => {
+  it('should render Text with default properties', () => {
```

Comment:
What sort of tests are we expecting our devs to write here? Same as all other tests that cover all cases? Wouldn't that be too much redundant effort? Any way we could run the web tests twice once with `render` from `renderWithSSR` and once with a normal `render`?

Reply:
nupe so these tests will be very minimal as there are no variations or click events on server so at the best you just want to render things on server to sanitise the dom doesn't breaks and renders without any issues to prevent unwanted usage of window and you can very well ignore this for simple components like badges/counters etc.